### PR TITLE
Demonstration mode to prevent password changes

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| Fixed tickets | 
| License       | MIT

### CODE REVIEW

File: src/Wallabag/CoreBundle/Controller/ConfigController.php [hunk 1]
```diff
@@ -57,13 +57,20 @@ public function indexAction(Request $request)
...
+                $userManager->updateUser($user, true);
+
+                $this->get('session')->getFlashBag()->add(
+                    'notice',
+                    'Password updated'
+                );
+            } else {
+                $this->get('session')->getFlashBag()->add(
+                    'notice',
+                    'In demonstration mode, you can`t change password.'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Controller/ConfigController.php]: What that `, can't you use \' ?

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Controller/ConfigController.php]: my fingers slipped

File: app/config/parameters.yml.dist [hunk 2]
```diff
@@ -45,3 +45,6 @@ parameters:
     fosuser_confirmation: true
 
     from_email: no-reply@wallabag.org
+
+    # set to true to prevent changing password
```
Comment (j0k3r) [inline app/config/parameters.yml.dist]: I would rather put :
> this parameter is only used for the wallabag public demo, it means user won't be able to change the password of the default account

Comment (nicosomb) [inline app/config/parameters.yml.dist]: :+1: you're right.

Comment (nicosomb) [inline app/config/parameters.yml.dist]: or not in fact. User**s** won't be abloe to change password of all accounts.

**Comment by tcitworld** [comment]
Wait, this means we can't modify passwords for user accounts on v2.wallabag.org. It should be only the `wallabag` account that should be blocked.

**Comment by nicosomb** [comment]
v2.wallabag.org is just a demo platform ...

**Comment by tcitworld** [comment]
Yeah but we did kind of push people to subscribe, even if we warned them it was only testing. Furthermore, maybe administrators of wallabag instances may want to propose a demo account to their customers.

**Comment by j0k3r** [comment]
You mean, it could more be a role instead ?

**Comment by tcitworld** [comment]
Or could we have a second parameter : the username of the demo account (I guess one is enough ?)

**Comment by nicosomb** [comment]
> Or could we have a second parameter : the username of the demo account (I guess one is enough ?)

Maybe the easier solution?

**Comment by j0k3r** [comment]
I'm wondering. Should this configuration be enable only using internal config instead? End user won't need to define this information when installing the app.
So moving the config in CraueConfig might be easier to test.

**Comment by nicosomb** [comment]
:+1:

**Pull request merged**